<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightForm</class>
 <widget class="QWidget" name="LightForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Lights &amp; Materials</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>414</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cmdClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabLight">
      <attribute name="title">
       <string>Lights</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>16</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cboLight">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Light:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Use toolbar to move lights.</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinLightR">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinLightB">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinLightG">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="labLightPos">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(0,0,0)</string>
       </property>
      </widget>
      <widget class="QSlider" name="sliderLightR">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>371</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderLightG">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>371</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderLightB">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>371</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkLightSync">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>110</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabMaterial">
      <attribute name="title">
       <string>Materials</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ambient Reflectivity:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Diffuse  Reflectivity:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Specular Reflectivity:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Specular Shininess:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatShininess">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>370</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatAmbientR">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatAmbientG">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatAmbientB">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatDiffuseR">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatDiffuseB">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatDiffuseG">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatSpecularG">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>310</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatSpecularB">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>340</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinMatSpecularR">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>280</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QComboBox" name="cboMaterial">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Material:</string>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatAmbientR">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatAmbientG">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatAmbientB">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatDiffuseR">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>180</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatDiffuseG">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>210</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatDiffuseB">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>240</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatSpecularR">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>280</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatSpecularG">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>310</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="sliderMatSpecularB">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>340</y>
         <width>301</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkMatAmbientSync">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>100</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkMatDiffuseSync">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>210</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkMatSpecularSync">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>310</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboLight</tabstop>
  <tabstop>spinLightR</tabstop>
  <tabstop>spinLightG</tabstop>
  <tabstop>spinLightB</tabstop>
  <tabstop>cboMaterial</tabstop>
  <tabstop>spinMatAmbientR</tabstop>
  <tabstop>spinMatAmbientG</tabstop>
  <tabstop>spinMatAmbientB</tabstop>
  <tabstop>spinMatDiffuseR</tabstop>
  <tabstop>spinMatDiffuseG</tabstop>
  <tabstop>spinMatDiffuseB</tabstop>
  <tabstop>spinMatSpecularR</tabstop>
  <tabstop>spinMatSpecularG</tabstop>
  <tabstop>spinMatSpecularB</tabstop>
  <tabstop>spinMatShininess</tabstop>
  <tabstop>cmdClose</tabstop>
  <tabstop>sliderLightG</tabstop>
  <tabstop>sliderLightB</tabstop>
  <tabstop>sliderLightR</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>sliderMatAmbientR</tabstop>
  <tabstop>sliderMatAmbientG</tabstop>
  <tabstop>sliderMatAmbientB</tabstop>
  <tabstop>sliderMatDiffuseR</tabstop>
  <tabstop>sliderMatDiffuseG</tabstop>
  <tabstop>sliderMatDiffuseB</tabstop>
  <tabstop>sliderMatSpecularR</tabstop>
  <tabstop>sliderMatSpecularG</tabstop>
  <tabstop>sliderMatSpecularB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
